.github/workflows/qa.yaml

```diff
@@ -0,0 +1,39 @@
+name: qa
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  qa:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v4
+
+      - name: PHP Setup
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+
+      - name: Validate composer.json and composer.lock
+        run: composer validate --strict
```

Comment:
We probably don't need this in `.github/workflows/test.yaml`, too, right?

Reply:
well, i thought that might make sense, as this job has it's own composer install and will fail, wenn the `composer.json` is corrupt.

But, to be honest: I think the step is completely useless :D Should we remove it completely?

Reply:
It's kind of a linting of the config. Thus, I think we only need to do it once. And I don't think it's useless because I think it's good to have a working composer setup. Why do you think it's completely useless?